Tests/Svg.UnitTests/ImageComparisonTest.cs

```diff
@@ -53,14 +59,16 @@ public void CompareSvgImageWithReference(ImageTestDataSource.TestData testData)
             {
                 basePath = Path.GetDirectoryName(basePath);
             }
-            basePath = Path.Combine(Path.Combine(basePath, "Tests"), "W3CTestSuite");
-            var svgBasePath = Path.Combine(basePath, "svg");
+            // var svgBasePath = Path.Combine(basePath, "svg");
             var baseName = testData.BaseName;
             bool testSaveLoad = !baseName.StartsWith("#");
             if (!testSaveLoad)
             {
                 baseName = baseName.Substring(1);
             }
+            var testsRoot = Path.Combine(basePath, "Tests");
+            basePath = TestsUtils.GetPath(testsRoot, baseName);
+            // basePath = Path.Combine(Path.Combine(basePath, "Tests"), "Issues");

```

Comment:
Original Code
```cs
45        public void CompareSvgImageWithReference(ImageTestDataSource.TestData testData)
46        {
47            // W3C Test Suites use external references to local fonts
48            SvgDocument.ResolveExternalXmlEntites = ExternalType.Local;
49            SvgDocument.ResolveExternalElements = ExternalType.Local;
50
51            var basePath = testData.BasePath;
52            while (!basePath.ToLower().EndsWith("svg"))
53            {
54                basePath = Path.GetDirectoryName(basePath);
55            }
56            basePath = Path.Combine(Path.Combine(basePath, "Tests"), "W3CTestSuite");
57            var svgBasePath = Path.Combine(basePath, "svg");
58            var baseName = testData.BaseName;
59            bool testSaveLoad = !baseName.StartsWith("#");
60            if (!testSaveLoad)
61            {
62                baseName = baseName.Substring(1);
63            }
64            var svgPath = Path.Combine(Path.Combine(basePath, "svg"), baseName + ".svg");
65            var pngPath = Path.Combine(Path.Combine(basePath, "png"), baseName + ".png");
66            CompareSvgImageWithReferenceImpl(baseName, svgPath, pngPath, testSaveLoad);
67        }
```
- svgBasePath of line 57 is not used so commented out
- The added lines in my codes (69-70) are updates to the line 56 (since we now have two directories `Issues` and `W3CTestSuite`).
- Line 69 creates the root tests directory path (`..\SVG\Tests`).
- Line 70 gets the directory of test file based on the file name as files in `Issues` starts with `__`
- So instead of just `basePath = Path.Combine(Path.Combine(basePath, "Tests"), "W3CTestSuite");` it will produce (depending on the file name):
```cs
basePath = Path.Combine(@"..\SVG\Tests", "Issues");
// -- or --
basePath = Path.Combine(@"..\SVG\Tests", "W3CTestSuite");
```

Reply:
Right, thanks!